src/data/popsicle.yml

```diff
@@ -0,0 +1,49 @@
+Type: desktop-application
+ID: popsicle.desktop
+Package: popsicle-gtk
+Name:
+  C: Popsicle
+Summary:
+  C: Flash multiple USB devices in parallel
+Description:
+  C: >-
+    <p>Write an ISO or other image to multiple USB devices all at once. Easily preparing a bunch of flash drives of your favorite OS with just a couple of clicks.</p>
+    <ul>
+      <li>Supports USB 2 and 3 devices</li>
+      <li>Use USB hubs for massively parallel writing</li>
+      <li>Verify your image with the SHA256 or MD5 checksum</li>
+      <li>Check the progress, speed, and success of each device while flashing</li>
+      <li>Open ISO or IMG files from the app, or straight from your file manager</li>
+    <ul>
+DeveloperName:
+  C: System76
+ProjectLicense: MIT
+Categories:
+  - System
+Keywords:
+  - USB
+  - Flash
+  - Drive
+  - Popsicle
+Url:
+  homepage: https://github.com/pop-os/popsicle
+Icon:
+  cached:
+    - name: popsicle.png
+      width: 64
+      height: 64
+    - name: popsicle.png
+      width: 128
+      height: 128
+Screenshots:
+  - thumbnails: []
+    source-image:
+    url: https://raw.githubusercontent.com/pop-os/popsicle/master/demo.gif
```

Comment:
afaik GIFs aren't supported in AppStream

Reply:
Yeah, just PNG or JPG https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Application.html#tag-dapp-screenshots

Reply:
Please use PNG